frontend/app/openproject-app.js

```diff
@@ -28,9 +28,12 @@
 
 var I18n = require('./vendor/i18n');
 
-// standard locales
-I18n.translations.en = require("locales/js-en.yml").en;
-I18n.translations.de = require("locales/js-de.yml").de;
+// load all js locales
+var requireExtraLocale = require.context('../../config/locales', false, /js-[\w|-]{2,5}\.yml$/);
```

Comment:
@NobodysNightmare I agree and will comply. 
@myabc What do you think about performance issues? Does Webpack solve our problems?